lib/pbench/server/api/resources/upload_api.py

```diff
@@ -79,26 +79,33 @@ def put(self, controller):
         md5sum = request.headers.get("Content-MD5")
 
         if not self.allowed_file(filename):
-            self.logger.debug(
+            self.logger.warning(
                 f"Tarfile upload: Bad file extension received for file {filename}"
             )
             abort(400, message="File extension not supported. Only .xz")
 
+        content_length = request.headers.get("Content-Length")
+        if not content_length:
+            self.logger.warning(
+                f"Tarfile upload: No Content-Length header value found for file {filename}"
+            )
+            abort(400, message="Missing required content-length header")
+
         try:
             content_length = int(request.headers.get("Content-Length"))
         except ValueError:
-            self.logger.debug(
+            self.logger.warning(
                 f"Tarfile upload: Invalid content-length header, not an integer for file {filename}"
             )
             abort(400, message="Invalid content-length header, not an integer")
         except Exception:
-            self.logger.debug(
-                f"Tarfile upload: No Content-Length header value found for file {filename}"
+            self.logger.exception(
+                f"Tarfile upload: Exception on interpreting Content-Length header value"
             )
-            abort(400, message="Missing required content-length header")
+            abort(HTTPStatus.INTERNAL_SERVER_ERROR, message="INTERNAL ERROR")
```

Comment:
What is our proposed practice with regard to HTTP status values:  numeric literals or symbolics?...I don't think we should be using both.  If we prefer symbolics, then we should be making an effort to replace the existing numerics, and we certainly shouldn't be adding new ones (e.g., line 92).

Reply:
I prefer symbolic names too but I think that can be addressed as a separate PR to map all the existing error codes from our code to their symbolic names.

Reply:
@npalaska, can you file an issue to track for the future?

Reply:
I concur that a mass conversion from numeric literals to symbolic references should not be done in this PR.  However, any _new_ code should use symbolic references...including the line above.  :-)

Reply:
It is now already addressed in PR https://github.com/distributed-system-analysis/pbench/pull/2287. So this commit is just for standalone db init now

Reply:
Also @portante I filed an issue https://github.com/distributed-system-analysis/pbench/issues/2310 to track for converting numerical http status codes to their corresponding symbolic names